expts/hydra-configs/finetuning/custom.yaml

```diff
@@ -0,0 +1,101 @@
+# @package _global_
+
+defaults:
+  - override /tasks/loss_metrics_datamodule: finetune
+
+constants:
+  benchmark: custom
+  task: finetuning_example-cls # finetuning_example-cls OR finetuning_example-reg
+  task_type: cls  # cls OR reg
+  data_path: expts/data
+  wandb:
+    name: finetune_${constants.task}
+    project: finetuning
+    entity: valencelabs
+    tags:
+    - finetuning
+    - ${constants.task}
+    - ${finetuning.pretrained_model}
+  seed: 42
+  max_epochs: 20
+  raise_train_error: true
+  model_dropout: 0.
+
+datamodule:
+  args:
+    batch_size_training: 256
+    batch_size_inference: 256
+    dataloading_from: ram
+    persistent_workers: true
+    num_workers: 8
+    
+    task_specific_args:
+      finetune:
+        df: null
+        df_path: ${constants.data_path}/${constants.task}/raw.csv
+        splits_path: ${constants.data_path}/${constants.task}/split.csv
+        smiles_col: smiles
+        label_cols: target
+        task_level: graph
+        epoch_sampling_fraction: 1.0
+  
+trainer:
+  model_checkpoint:
+    save_top_k: 0
+    dirpath: ~/project/outgoing/finetuning/${constants.task}/${now:%Y-%m-%d_%H-%M-%S.%f}/
+    every_n_epochs: 200
+    save_last: false
+  trainer:
+    precision: 32
+    check_val_every_n_epoch: 1
+    accumulate_grad_batches: 1
+  
+predictor:
+  optim_kwargs:
+    lr: 0.000001
+  torch_scheduler_kwargs:
+    module_type: WarmUpLinearLR
+    max_num_epochs: ${constants.max_epochs}
+    warmup_epochs: 3
+    verbose: False
+
+
+
+# == Fine-tuning config == 
+
+finetuning:
+  task: finetune
+  level: graph
+  pretrained_model: dummy-pretrained-model
+  finetuning_module: graph_output_nn  
+  sub_module_from_pretrained: graph
+  new_sub_module: graph
+  drop_depth: 1
+  added_depth: 1
+  new_out_dim: 1024 # TODO: infer automatically
```

Comment:
Are you finetuning on 1024 tasks at the same time? Or is that the hidden dim?

Perhaps replacing the `TODO` by `# Change according to the number of finetuning tasks`